sdk/python/runtime/main.go

```diff
@@ -13,11 +13,15 @@ func New(
 ) *PythonSdk {
 	return &PythonSdk{
 		SDKSourceDir: sdkSourceDir,
+		RequiredPaths: []string{
+			"**/pyproject.toml",
```

Comment:
It will yeah, but I think what we'd optimally want is the "nearest" one in any parent dir in the context directory (I'm assuming `pyproject.toml` has similar-ish behavior to `go.mod`/`package.json`/etc.). To do that we'll need even more intelligence.

My first thought was something like actually giving this SDK function `ModuleSource` so the SDK can inspect it. The problem there is we are lazily loading these dirs, so we actually couldn't give SDKs plain old `Directory`, we'd need to give them some object that lets them stat + read files from the caller's host. Which is do-able technically but too much for here.

There's probably other ways of approaching this like allowing the SDK to return something that conveys i.e. "give me any file with this name from the nearest parent dir above the source dir" and then let the engine do the rest.

Either way, all that felt like excessive optimization for now, so I just went with this approach. Can optimize more in the future as needed.

Reply:
Since all files from source dir are already included, I think I'll remove this until a use case presents itself. To be useful, it'd have to be configurable by the modules.

Reply:
Cool sounds good, I do see now that there's no subtest under `TestModulePythonInit` like `respects parent pyproject.toml`, so I guess it's different than go/ts right now